timjroberts/cucumber-js-tsflow
# Fix compiling errors

We the latest configuration in `master`, it is not possible to build/run the built-in tests.  I was able to resolve the errors with a couple of small changes:

1) Running "`npm update`" in both the root and cucumber-tsflow folders
2) Updating the import of "@cucumber/messages" in `cucumber-tsflow/src/binding-decorator.ts` to match recent changes in cucumber's architecture

Note: the existing pre-commit hook injected a couple of formatting changes elsewhere in cucumber-tsflow/src/binding-decorator.ts - these changes are purely cosmetic and have no functional effect on the codebase.

Steps to test:
1) Perform a clean checkout from  https://github.com/timjroberts/cucumber-js-tsflow (e.g.: "git reset --hard origin/master")
2) Attempt to run commands "npm install" and "npm run build"
   - Test code cannot start; tsc reports a compiling error
3) Check out the branch for this PR: https://github.com/zcsteele/cucumber-js-tsflow/tree/update-binding-decorator
4) Run "npm install" and observe the tests completing successfully
